doc-content/jbpm-docs/src/main/asciidoc/BAM/DashbuilderRuntime-exporting-importing-dashbuilder-data-proc.adoc

```diff
@@ -0,0 +1,193 @@
+[id='DashbuilderRuntime-exporting-importing-dashbuilder-data-proc_{context}']
+= Exporting and importing Dashbuilder data
+
+ifdef::PAM,JBPM[]
+
+Dashbuilder is a dashboard and reporting tool integrated in {CENTRAL} and is used by the Datasets editor and Content Manager page. There are three data types:
+
+* Datasets
+* Pages
+* Navigation
+
+You can import and export Dashbuilder data as ZIP files in {CENTRAL}.
```

Comment:
Trivial, but here and in the DM version, you say "can import and export" but then the order is reversed in the actual sections, first exporting then importing. I try to be consistent between how I summarize/introduce content and how I then present it. Just helps users imo.

Reply:
True! I have introduced my "export" module first. I changed it accordingly in the content. 
I really liked this observation and will keep this in my mind going forward.